### ISSUE TITLE
Владелец карты доступа

### ISSUE THREAD

**Issue by koldun212** [JustMarfix/VeryUsualDay#77]
Когда человеку **выдаётся** карта, она закрепляется за его именем, т.е если другой человек её подберёт,  то он увидит кому она принадлежит

Пример текста, когда человек достаёт карту:

Ключ карта доступа -  _Сотрудник(Охранник/Рабочий/ГОР/Агент ОВБ)_  - номер
Идентификационный номер - 5-6 чисел, возможно и буквы

**Comment by JustMarfix** [comment]
С новым функционалом мы сможем сделать это нормально

**Comment by koldun212** [comment]
да, знаю, там просто нужно настраивать будет всё сильно, в ближайшее время займусь и тебе скину всё
